aws/resource_aws_secretsmanager_secret.go

```diff
@@ -0,0 +1,316 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+
```

Comment:
whitespace

Reply:
`goimports` strikes again! Not sure why it thinks it needs to do this sometimes. I'll get it updated